qa/rpc-tests/test_framework/test_framework.py

```diff
@@ -186,7 +186,7 @@ class ComparisonTestFramework(BitcoinTestFramework):
 
     def __init__(self):
         super().__init__()
-        self.num_nodes = 2
+        self.num_nodes = 1
```

Comment:
There was a potential issue where if we backported changes from upstream that included tests assuming the previous setting. However, upstream removed this framework entirely in https://github.com/bitcoin/bitcoin/pull/11818 (merged in 2018), so it's unlikely there are additional tests that could be backported, and if there are, we can handle the issue ourselves (and it would be detectable because the test would try to access a node that doesn't exist).

Reply:
I also notice that one reason for removing the comptool code was because it was unused:

https://github.com/bitcoin/bitcoin/issues/10603#issue-236262893

Don't we have an interesting use case where `zebrad` is one of the nodes and `zcashd` another?